docs/topics/settings.rst

```diff
@@ -431,6 +431,24 @@ parameter at init (this is the ``OpenSSL.SSL`` method mapping
 
 .. setting:: DOWNLOADER_CLIENT_TLS_METHOD
 
+DOWNLOADER_CLIENT_TLS_CIPHERS
+-----------------------------
+
+Default: ``'DEFAULT'``
```

Comment:
I wonder if we should use a list of ciphers which browsers use as a default, instead of using OpenSSL defaults, and provide another default value ("OPENSSL_DEFAULT"). This is a larger task, and I have no idea where to get these defaults, but it could make Scrapy work as expected in more cases. AFAIK in OpenSSL defaults are really for the server side, these defaults are more restrictive than browsers'. 

Also, if I recall correctly, some of the ciphers which browsers accept are disabled at compile time in OpenSSL, which is an additional challenge.

Reply:
Aha, DEFAULT actually came from OpenSSL, so let's keep it as-is. Browser-like defaults is a separate feature, let's first get DOWNLOADER_CLIENT_TLS_CIPHERS merged. 

My worry was that we may have to change "DEFAULT" to something else if we change the actual DOWNLOADER_CLIENT_TLS_CIPHERS default value to a browser-like, and so we should think about it now, but as "DEFAULT" comes from OpenSSL, it is better to keep it as-is.